src/goto-instrument/code_contracts.cpp

```diff
@@ -549,22 +549,23 @@ bool code_contractst::apply_function_contract(
   return false;
 }
 
-void code_contractst::apply_loop_contract(
-  const irep_idt &function_name,
-  goto_functionst::goto_functiont &goto_function)
+void code_contractst::apply_loop_contracts()
```

Comment:
I realise this is a little pedantic but I think the pattern of having a `goto_programt` and a `goto_functionst` version of functions, even if `apply_loop_contracts()` is simply a loop that calls `code_contractst::apply_loop_contract(const irep_idt &function_name,  goto_functionst::goto_functiont &amp;goto_function)`.

Reply:
I kept this original function and added `apply_loop_contracts()` that just loops over all goto functions and calls `add_loop_contract` on each one.